canonical-websites/www.ubuntu.com
# Updated pricing page and related pages

## Done

- Updated all text and tables on /support/plans-and-pricing
- Added variables for ticks to not have to keep typing the tick image
- Replaced all *.html pages tick image tags

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1FS1slJOfWXLmPob1eXrQ32DU4s6WP4oyGoT3t5E4W8Y/edit#)

## Issue / Card

Fixes #4789
